recipe/meta.yaml

```
@@ -9,7 +9,7 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 8
+  number: 9
   script: python setup.py install --single-version-externally-managed --record=record.txt
```

[Conda-forge docs](https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#optional-bld-bat-and-or-build-sh) suggest using pip instead:
```suggestion
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```